Add Troubleshooting Steps for chocolateyguicli Command (#394)
chocolatey/docs: input/en-us/chocolatey-gui/known-issues.md

If you are running into any issues with using the Chocolatey GUI extension alongside Chocolatey GUI, please ensure that you are using at least Chocolatey GUI v0.17.2 and also using the latest pre-release version of Chocolatey GUI extension which is currently `0.1.0-GL-4-20200713`.

## Chocolatey GUI asking for credentials for my non-administrator accounts

If a user is a member of the Built-in AD group `Network Configuration Operators`, then that means they have an elevation token available and will be treated in the same way as administrative accounts. To fix this, you have two options:

* Remove the users from `Network Configuration Operators` - PowerShell offers an alternative to `ipconfig /flushdns` that does not require admin permissions - `Clear-DnsClientCache`.
* OR change the ChocolateyGui.exe.manifest file in v0.17.0+ in the Chocolatey GUI folder under Program Files to `<requestedExecutionLevel level="asInvoker" uiAccess="false" />` (from `highestAvailable`).

Please see the following [GitHub Issue](https://github.com/chocolatey/ChocolateyGUI/issues/629) for more details.

## Pin command does not work when running with Background Mode enabled

> In the current versions of Chocolatey CLI (v0.11.1) and Chocolatey GUI (v0.19.0), when attempting to pin a package using the Chocolatey CLI (choco.exe) with Background Mode enabled, things will work as expected.  i.e. when running:
> 

Reviewer: ```suggestion
In the current versions of Chocolatey CLI (v0.11.1) and Chocolatey GUI (v0.19.0), when attempting to pin a package using the Chocolatey CLI (choco.exe) with Background Mode enabled, things will work as expected.  i.e. when running `choco pin add --name="procmon"`.

```

> ```powershell
> choco pin add --name="procmon"
> ```

Reviewer: ```suggestion
```

>
> Exception: System.NullReferenceException: Object reference not set to an instance of an object

This will be addressed in a future version of Chocolatey CLI/Chocolatey GUI.

## chocolateyguicli Command Not Registered by System

When recieveing an error in shell saying the chocolateyguicli is not registered or `ObjectNotFound`. Please run through the following troubleshooting steps.

Step 1: Reinstall the chococlateygui package. `choco upgrade chocolateygui -y --force`

> Step 2: If a re-install of the package did not fix the issue. Please check if you are running an AV solution, that it is not blocking the write action being taken on the Chocolatey GUI Config. Sometimes AV solutions will block this request from happening.

Reviewer: ```suggestion
1. If step 1 did did not fix the issue, please check that your antivirus software is not blocking the Chocolatey GUI Config file from being written to.
```

Reviewer: Can you add the location where the Chocolatey GUI config is? So:

```
1. If step 1 did did not fix the issue, please check that your antivirus software is not blocking the Chocolatey GUI Config file at <PATH TO CONFIG FILE> from being written to.
```
